src/main/java/org/fundacionjala/coding/reinaldoA/movies/Customer.java

```diff
@@ -0,0 +1,49 @@
+package org.fundacionjala.coding.reinaldoA.movies;
+
+import java.util.Enumeration;
+import java.util.Vector;
+/**
+ * customer class
+ **/
+class Customer {
+    private String _name;
+    private Vector _rentals = new Vector();
+
+    public Customer(String name) {
+        _name = name;
+    }
+
+    public void addRental(Rental arg) {
+        _rentals.addElement(arg);
+    }
+
+    public String getName() {
+        return _name;
+    }
+
+    public String statement() {
```

Comment:
this method is doing 3 things
it is calculating the totalAmount, totalFrequentRenterPoints and is printing.
please split the method